dht/handler.go

```diff
@@ -99,3 +99,11 @@ func (dht *DHT) pongRequestHandler() {
 		}
 	}
 }
+
+func (dht *DHT) republishRequestHandler() {
+	for {
+		request := <- dht.nw.RepublishRequestCH()
```

Comment:
You want to use this method instead:
```go
dht.db.LocalItemCh()
```

And then do a:
```go
dht.Put(value)
```

With the value from the channel.

Reply:
Correct! :smile: